### ISSUE TITLE
Move / transfer this project to an own GitHub Organization

### ISSUE THREAD

**Issue by MarkusTiede** [junaid33/opensource.builders#222]
I'd like to propose to move this **awesome** project to a more neutral / meritocratic location : an own GitHub Organization with more members / committer being able to participate.

I've created https://github.com/open-organization a while ago - with a very similar vision / idea in mind: the proposal would be to move there and allow access for anyone who's interested.

Please let me know what you all think - thanks and with best regards - looking forward to hearing back from you!

**Comment by MarkusTiede** [comment]
Also I think that the current approach (maintaining this by one individual) won't scale well.

**Comment by jcklpe** [comment]
@MarkusTiede I recently tagged @junaid33 and he added me to this repo. He might be interested in doing the same for you, idk. 

I have no opinion about putting stuff to a github org or not. Right now I just see this as a repo I can help with. 

I think it's worth pointing out there are other websites of a similar nature to opensource.builders such as alternativeto.net or slant.co though neither of those have the website code or content managed through open source means. I think that gives opensource.builders a huge advantage for scaling (which I think you in part point to).

**Comment by MarkusTiede** [comment]
I agree - doing the open from the beginning is a big advantage: thanks for sharing that in this manner!

In my (humble) opinion having this in a more neutral space could be useful.

**Comment by junaid33** [comment]
@MarkusTiede I can make you a collaborator if you'd like. I would like to keep the repo under my name since it is a sort of portfolio website for me.

**Comment by MarkusTiede** [comment]
Sure that’s understandable: I’ll put a fork to that orga for now.

Please feel free to add me: is there anything in particular that one could help with at the moment?

**Comment by junaid33** [comment]
Sure some short term issues are:

1. Add pagination - if we add more comparisons, the website will eventually have to load a huge JSON file and that will slow it down
2. Self-host images - the images are being linked from different sources and if those sources go down, the images will not show
3. Add categories - currently we have comparisons from social media to developer tools, filtering thru categories will help users find what they need quickly
4. Add a filter for categories, license, language - we're using js-search but I cannot figure out how to add multiple filters without creating 1 long string
5. Add a sign-in with Github button so users can upvote and request right on the website instead of having to go on Github.

Can you help with any of these?

**Comment by NicoleTiede** [comment]
Hi @junaid33 - rather than implementing I'd like to help triaging / organizing; for now I've started searching for the corresponding (and if not found file new ones) issues from 1-5:
 - 1 -> #245 
 - 2 -> #109
 - 3 -> #246
 - 4 -> #247
 - 5 -> #248

**Comment by MarkusTiede** [comment]
Hi @junaid33 - rather than implementing I'd like to help triaging / organizing; for now I've started searching for the corresponding (and if not found file new ones) issues from 1-5:

1 -> #245
2 -> #109
3 -> #246
4 -> #247
5 -> #248
